Containerfile.agent

```diff
@@ -6,7 +6,7 @@ RUN mkdir /app/www && \
      tar xf /tmp/agent-latest.tgz -C /app/www
 
 # Builder container
-FROM registry.access.redhat.com/ubi9/go-toolset as builder
+FROM --platform=linux/amd64 registry.access.redhat.com/ubi9/go-toolset as builder
```

Comment:
If you will send this as separate PR, we can merge ASAP and get this PR smaller.

Reply:
Hi @machacekondra ,
I opened a different PR for this
https://github.com/kubev2v/migration-planner/pull/204
I will rebase this PR after it merged